src/app/explorer/resource_list.tsx

```diff
@@ -105,9 +105,7 @@ export default function ResourceList(props: ResourceListProps) {
     return (
         <div>
             <h1>{props.resource.singular_name}</h1>
-            <Link to={"_create"}>
-                <span>Create</span>
-            </Link>
+            <Link to={"_create"} className={buttonVariants({ variant: "default" })}>Create</Link>
```

Comment:
Nit: I personally prefer the nested approach, since it doesn't require forcing a link to look like a button, and makes the codebase easily searchable for buttons (if you search "<Button", this won't show up in there): 

```
<Button asChild>
  <Link href="/login">Login</Link>
</Button>
```